app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars_list: list[Car]) -> float:
+        total_income = 0
+        for car in cars_list:
+            if car.clean_mark < self.clean_power:
+                wash_price = self.calculate_washing_price(car)
+                total_income += wash_price
```

Comment:
```suggestion
                total_income += self.calculate_washing_price(car)
```
this would be better, and you don't need this redundant variable